api/tests/MemberMatch/MemberMatchTests.cs

```diff
@@ -326,5 +329,156 @@ public void TestMethod9()
             Assert.AreEqual(probability, 0.8693, delta: 0.0001);
         }
 
+        [TestMethod]
+        public void TestBig1()
+        {
+            // Download from https://1drv.ms/u/s!AkoPP4cC5J64xMgga4595XA390eN5Q?e=neugfs
+            string root = @"D:\OneDrive\Shares\RaceResults";
+            if (!File.Exists(root))
+            {
+                Trace.WriteLine("Skipping 'TestBig1'");
+                return;
+            }
+
+            var scorer = new Scorer(root + "/name_probability.tsv");
+            var members = new Members(root + "/sample_members.tsv");
+
+            foreach (var (withCity, filename, minimumScore, expectedFile) in new[]
+            {
+                                (true, "/sample_results_withcity.txt", -9.0, "/expected_withcity.txt"),
```

Comment:
To keep test results separated, it's good to not have a for loop in the test method.

I think this test is a great candidate for a parameterized test: https://makolyte.com/csharp-parameterized-tests-with-mstest-v2/

Reply:
Cool. the next pull request will include this.